distributed/comm/core.py

```diff
@@ -202,6 +202,9 @@ def _raise(error):
         )
         raise IOError(msg)
 
+    # don't spam the server with connect attempts
+    backoff = min(0.1, timeout/20 if timeout else 9999)
```

Comment:
In the meantime @smurfix, this is causing a lint failure: https://travis-ci.org/dask/distributed/jobs/652056913#L496.

Reply:
Previously, the first attempt would occur 0.01s after the previous attempt. Should the first element in the min be 0.01?

Reply:
I'll fix the lint failure.

Yes, the previous interval was 0.01; the question is, how fast would the service (re)start? I assume that it will take a non-negligible fraction of second, thus retrying almost-immediately is not going to improve things.

Reply:
What scenarios result in an EnvironmentError? Could something like a temporary network hiccup cause that? If so, I'd think retrying almost immediately is appropriate.

Reply:
I've gone ahead and set the minimum to 0.01